Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreeTransaction.h

```diff
@@ -248,6 +252,7 @@ class RemoteLayerTreeTransaction {
     friend struct IPC::ArgumentCoder<RemoteLayerTreeTransaction, void>;
 
     WebCore::PlatformLayerIdentifier m_rootLayerID;
+    WebCore::ProcessIdentifier m_processIdentifier;
```

Comment:
Do we really need to IPC this over on every transaction? Don't we know which process we're getting messages from?

Reply:
On Cocoa platforms we can get a pid_t of the connected process from the Connection, but pids are reused and the non-platform-specific Connection does not yet have a way to get the ProcessIdentifier of the connected process.  Maybe I could add that.